### DISCUSSION TITLE
Sign up error

### DISCUSSION THREAD

**Discussion by Waptech07** [vercel/vercel#10615] [General]
### Description

I created an account on Vercel, then I tried to login but I keep getting this message "We are unable to complete your signup. Please read [this guide] for further details.". 
![image](https://github.com/vercel/vercel/assets/112312221/eb064078-0d91-4c80-8324-e7f968e49a1c)

**Comment by blackrivercoffee** [comment]
I'm having the same problem, even though I sent an e-mail, I still haven't received a response.

**Reply by Waptech07** [reply]
No response yet too

**Reply by blackrivercoffee** [reply]
Yes, my membership is still not approved.
